sw/airborne/modules/nav/nav_rotorcraft_hybrid.c

```diff
@@ -321,6 +333,40 @@ static void nav_hybrid_circle(struct EnuCoor_f *wp_center, float radius)
   nav.setpoint_mode = NAV_SETPOINT_MODE_SPEED;
 }
 
+/* Function that estimates at every completion of a circle if the airspeed measured is correct*/
```

Comment:
Would it be possible to make this a separated module ? Checking consistency of airspeed measurement is nice but should not be part of the navigation code I think.